openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Ok. The problem is that this code breaks, so it cannot stay here as we now have automatic load the variables in python file.
Also, I think it would be nice not to have dead code or non-functional code in progress in the main repository ;)